dev/plugins/hu.elte.txtuml.export.javascript/src/hu/elte/txtuml/export/javascript/resources/res/main.css

```diff
@@ -24,3 +24,404 @@
+	    -ms-flex-pack:justify;
+	        justify-content:space-between;
+	-webkit-box-align: center;
+	    -ms-flex-align: center;
+	        align-items: center;
+}
+
+.button{
+	padding: 0.5em;
+	border-radius: 5px;
+	-webkit-transition: all 0.2s;
+	-o-transition: all 0.2s;
+	transition: all 0.2s;
+	text-align: center;
+	-webkit-user-select: none; /* Safari */        
+	-moz-user-select: none; /* Firefox */
+	-ms-user-select: none; /* IE10+/Edge */
+	user-select: none; /* Standard */
+	cursor: pointer;
+}
+
+.control{
+	background-color: #8da1a1;
+	border: outset 2px;
+}
+
+#zoom-in{
+	-webkit-box-flex: 1;
+	    -ms-flex: 1 0;
+	        flex: 1 0;
+}
+
+#zoom-out{
+	margin-left:0.2em;
+	-webkit-box-flex: 1;
+	    -ms-flex: 1 0;
+	        flex: 1 0;
+}
+
+#reset-view{
+	margin: 0.2em;
+}
+
+#toggle-pan{
+	margin: 0.2em;
+}
+
+#toggle-wheel-zoom{
+	margin: 0.2em;
+}
+
+.toggle-checkbox{
+	display:none;
+}
+
+.toggle-checkbox:checked ~ .label{
+	background:#99aacc;
+	border: outset 2px;
+}
+
+.control:hover{
+	background-color: #396dbc !important;
+	color:white !important;
+	text-shadow: 0 2px 2px black !important;
+}
+
+.control:active{
+	border: inset 2px !important;
+	background-color: #194b9a !important;
+}
+
+.switch{
+	overflow: hidden;
+	display: -webkit-box;
+	display: -ms-flexbox;
+	display: flex;
+	-webkit-box-orient: horizontal;
+	-webkit-box-direction: normal;
+	    -ms-flex-direction: row;
+	        flex-direction: row;
+	-ms-flex-pack: distribute;
+	    justify-content: space-around;
+	position: relative;
+	background-color: #8da1a1;
+	padding: 0;
+}
+
+.switch-content{
+	-webkit-box-flex: 1;
+	    -ms-flex: 1 1;
+	        flex: 1 1;
+	-webkit-transition: all .2s ease-out;
+	-o-transition: all .2s ease-out;
+	transition: all .2s ease-out;
+	padding: .5em;
+	border-radius: 5px;
+}
+
+.switch-border{
+	background: transparent;
+	width: 50%;
+    position: absolute;
+    z-index: 2;
+    height: 100%;
+    top: 0;
+    left: 0;
+    display:-webkit-box;
+    display:-ms-flexbox;
+    display:flex;
+    -ms-flex-pack: distribute;
+        justify-content: space-around;
+    -webkit-transition: all .2s ease-out;
+    -o-transition: all .2s ease-out;
+    transition: all .2s ease-out;
+    border-radius: 5px;
+    border:  2px solid white;
+}
+
+.switch-border:hover{
+	-webkit-box-shadow: inset 0 0 4px 2px #396dbc;
+	        box-shadow: inset 0 0 4px 2px #396dbc;
+	border:  2px solid #396dbc;
+}
+
+.toggle-checkbox:checked + .switch > .switch-right{
+	background-color: #20ac59;
+	text-shadow: 0 0 5px white;
+}
+
+.toggle-checkbox:checked + .switch > .switch-left{
+	background-color: #8da1a1;
+}
+
+.toggle-checkbox + .switch > .switch-left{
+	background-color: #20ac59;
+	text-shadow: 0 0 5px white;
+}
+
+.toggle-checkbox:checked + .switch > .switch-border{	
+    left: 50%;
+}
+
+.toggle-checkbox:checked + .toggle{
+	background-color: #20ac59;
+	text-shadow: 0 0 5px white;
+	border: 2px inset;
+}
+
+@media screen and (orientation: portrait) {
```

Comment:
The internal CSS editor of Eclipse cannot parse this line which results in errors inside the user workspace when the file is automatically imported after visualization. See #513. Last time we didn't find a better solution than removing the problematic block (see 74cbe2b), but as I can see that's not a viable option here.